src/components/AnalysisModal.js

```diff
@@ -162,11 +180,17 @@ export const AnalysisModal = Utils.withDisplayName('AnalysisModal')(
         }
       }, [img({ src: rstudioLogo, style: styles.image })]),
       div({
-        style: { opacity: currentApp ? '0.5' : '1', ...styles.toolCard }, onClick: () => {
+        style: { opacity: galaxyApp ? '0.5' : '1', ...styles.toolCard }, onClick: () => {
```

Comment:
Hmm... If I'm a user and I create a galaxy app, and then create a cromwell app what is the intended behavior?

It seems you've attempted to allow the two to co-exist, which is I think what we want.

That being said, if the Galaxy/Cromwell app is still creating, it allows me to attempt to create another app of a different type and the backend kicks back and says the K8 cluster is still creating. We may want to consider adding a check to see if ANY app is Precreating/Provisioning to the disabled flag here and within the context bar. Thoughts?

Reply:
I do think we want both to co-exist. I think that you shouldn't get the error, and instead it should just wait until the cluster is created and then proceed with the creation of the 2nd app. I will create a ticket for the problem.

The fact that Cromwell and Galaxy share a cluster is interesting… and perhaps something that we want to change. We have questions about the cluster lifecycle and the cost to the user when a cluster has been created, but apps have been deleted. Starting up a cluster takes a LONG time, so I anticipate users are going to want us to pre-create the cluster if at all possible.

Reply:
Yeah we've had discussions around this. The problem is when the cluster pre-creation occurs in the workspace lifecycle and cost. There's not really a good place besides directly when the users requests an app, since not all users will use apps, and we don't want them paying for the cluster premium before that point.

Reply:
Thanks @jdcanas for catching this!

I'd also agree that we want them to co-exist but would like to get UX/Product thoughts on it too. @cahrens, could you add us + Alessandro and Tiffany on the ticket you'll create so we can get on the same page.

> We have questions about the cluster lifecycle

Happy to chat about it 😊 

> the cost to the user when a cluster has been created, but apps have been deleted

We (`leonardo-cron-jobs`) detect when that happens and if it's been over a few hours, delete the cluster.

Reply:
I created https://broadworkbench.atlassian.net/browse/IA-3071